### PR TITLE
Added Communications

### PR DESCRIPTION
**Pull request by dcyoung-dev**
### Added a Comms log
To display all communications sent and received

### Added a Send Comms form
To send communications (this only adds to the log for now)

### Added a Fake Read form
To simulate a message being received from the Command Station (message added to logs as a "received" message)

### Screenshots
| page |  |
|---|---|
| Empty logs | Screenshot 2023-12-07 at 07-38-42 Vuetify 3 |
| Send form | Screenshot 2023-12-07 at 07-39-01 Vuetify 3 |
| Send form (user-input) | Screenshot 2023-12-07 at 07-40-12 Vuetify 3 |
| Logs (sent items) | Screenshot 2023-12-07 at 07-40-50 Vuetify 3 |
| Fake read form | Screenshot 2023-12-07 at 07-39-11 Vuetify 3 |
| Fake read form (user-input) | Screenshot 2023-12-07 at 07-41-32 Vuetify 3 |
| Logs (populated) | Screenshot 2023-12-07 at 07-41-41 Vuetify 3  |

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,6 @@
     },
     "dependencies": {
         "@mdi/font": "^7.3.67",
-        "@slipmatio/control-knob": "^0.1.0",
```
Comment (dexslab) [inline package.json:14]: Why are we removing the control knob are we planning on writing our own component for rotary knob

Comment (dcyoung-dev) [reply package.json:14]: It was causing errors with the components not being found. I think this is an issue with the package we were using. 
Also I'd really like to clean up the codebase and remove code that is not being used 👌 2 birds, 1 stone. 
https://github.com/DCC-EX/EX-WebThrottle2/pull/25/commits/29fa423317522ddeab9fa3616b3610ea6ebc2858

Comment (dcyoung-dev) [reply package.json:14]: p.s. see failing Playwright hobs on the `main` branch

Comment (dexslab) [reply package.json:14]: I knew they were failing i was still in the process of writing the component

**Review by dexslab — APPROVED** [review]

**Pull request merged**